app/backend/src/couchers/models.py

```diff
@@ -49,6 +49,16 @@ class SmokingLocation(enum.Enum):
     no = 4
 
 
+class Fluency(enum.Enum):
+    fluency_unspecified = 0
+    fluency_say_hello = 1
+    fluency_beginner = 2
+    fluency_intermediate = 3
+    fluency_advanced = 4
+    fluency_fluent = 5
+    fluency_native = 6
```

Comment:
I think naming it `LanguageFluency` would be clearer. Also, no need to prefix with `fluency_` in models. The other enums start from 1, would be good to stick to that (kalvdans is working to replace them with `enum.auto()`, but that's another PR).

Do we need to have 7 options? I'm not sure if this was discussed. I think unspecified, beginner, intermediate, and fluent/native would suffice?

Reply:
> Also, no need to prefix with fluency_ in models.

great point, I'll change that.

> Do we need to have 7 options? I'm not sure if this was discussed

It was discussed in https://github.com/Couchers-org/couchers/issues/409#issuecomment-743934151

Reply:
Just to note (though you probably realized this), for .proto enums, this would be correct - 0 is unspecified and there are prefixes